concent_api/core/tests/test_auth_accept_or_reject_results.py

```diff
@@ -62,14 +63,15 @@ def test_provider_forces_subtask_results_for_task_which_was_already_submitted_co
         )
 
         with mock.patch('core.message_handlers.base.is_provider_account_status_positive', _get_provider_account_status_true_mock):
-            with freeze_time("2018-02-05 10:00:30"):
-                response = self.client.post(
-                    reverse('core:send'),
-                    data                                = serialized_force_subtask_results,
-                    content_type                        = 'application/octet-stream',
-                    HTTP_CONCENT_CLIENT_PUBLIC_KEY      = self._get_encoded_provider_public_key(),
-                    HTTP_CONCENT_OTHER_PARTY_PUBLIC_KEY = self._get_encoded_requestor_public_key(),
-                )
+            with mock.patch('core.message_handlers.new_sci_rpc', self._pass_rpc_synchronization):
```

Comment:
If you we agree on moving sci initialization to sci_backend this mock can be removed, as well as further usages.

Reply:
Done